### DISCUSSION TITLE
How to make a snapshot of my doorbell by a trigger?

### DISCUSSION THREAD

**Discussion by canedje** [NRCHKB/node-red-contrib-unifi-os#65] [Q&A]
I just using the contrib.
It works nice.
I'm able to trigger on a "Door Bell Ring" make a action and  also taking a snapshot.
How can I take an other snapshot initiated/ triggert by an other flow or node? (for example after eacht 10 seconds or so?)
I tried by an inject node but then i get:  Invalid Payload

![image](https://github.com/NRCHKB/node-red-contrib-unifi-os/assets/23150947/d7616ab5-3231-4c41-8a53-ee04d0d51906)

Thanks

**Comment by crxporter** [comment]
So sorry for missing this.

If you want to just grab a picture every 10 seconds - you shouldn't use this node. Just grab it right from the camera!

Step 1: go to the camera local IP address and enable "anonymous snapshot" on the camera (log in using ubnt and your device password)
Step 2: use nodered to download the image from `http://camera.ip/snap.jpeg`

That will give you a jpeg file to do whatever you like with it.
